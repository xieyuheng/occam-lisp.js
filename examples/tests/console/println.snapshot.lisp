(println 123)
(println '(a b c))
(println "abc")
