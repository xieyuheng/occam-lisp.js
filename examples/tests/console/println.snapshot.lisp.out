123
['a 'b 'c]
"abc"
